content/compose/intro/features-uses.md

```diff
@@ -0,0 +1,68 @@
+---
+description: Key benefits and use cases of Docker Compose
```

Comment:
Really liked this page as a starting point, especially the What's next links at the end 👍